grammar.js

```diff
@@ -930,12 +930,12 @@ module.exports = grammar({
     ),
 
     string: $ => seq(
-      field('prefix', alias($._string_start, '"')),
-      repeat(choice(
-        field('interpolation', $.interpolation),
-        field('string_content', $.string_content)
-      )),
-      field('suffix', alias($._string_end, '"'))
+      field('open', alias($._string_start, '"')),
+      field('value', repeat(choice(
+        $.interpolation,
+        $.string_content
+      ))),
```

Comment:
You are overlooking an important use case for fields: negations.

Currently there is no way to look for the _absence_ of a named child. Thus, with your scheme there would be no way of querying for strings that do not contain interpolations. With fields, you can at least do `(string !interpolation)` (which in fact was the motivation for adding this field in the first place).

Reply:
Thank for the note, it's good point about negations but I'd suggest to remove tautology for field names, see the updated commit.
And leave a `prefix` field name for actual [string prefixes](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals) like `f`, `r`, `b`, etc.